### DISCUSSION TITLE
Set specific path diffrent Display mode

### DISCUSSION THREAD

**Discussion by Monirzadeh** [koreader/koreader#12191] [Q&A]
I have multiple directory and my default view is mosaic (for directory like Home Collection etc)
but i have some shortcut folder for example `/document/specific/path` that i keep other kind of documents,
if i set display mode to mosaic this specific path `/document/specific/path` will be mosaic too, but i want see this specific directory in for example  `Clasisc (filename only)`

What is use case?
i keep some kind ebook there (programmatically) that most of them not have good quality image or cover (and change ebooks so frequently) so the best view to disgusting between them is Classic 

but my homepage is the best view in Mosaic with cover images

can i set different `Display Mode` for specific path?

**Comment by poire-z** [comment]
No, there's no support for that.
I think it would be very complicated to implement - we currently "overload" the FileBrowser with specific Mosaic/List hacks when FileBrowser is started, switching that while it is running as you browse up and down directories would be complicated/risky.

**Reply by Monirzadeh** [reply]
thanks. as not prefect alternative but is it possible to https://github.com/koreader/koreader/discussions/12191#discussioncomment-10125053 for now?

**Comment by jonnyl2** [comment]
Just in case you haven't updated yet or read the release notes, there is now support for changing display modes and sort by-options via gestures, for super convenient switching. I suggest creating profiles so you can change style and sorting simultaneously.

**Reply by Monirzadeh** [reply]
thanks.
it is not prefect but work. i should set different gesture for switch between two display mode.
is it possible to toggle between two display mode with one gesture?

**Reply by hius07** [reply]
> switch between two display mode

Maybe a user patch for
https://github.com/koreader/koreader/blob/5affb68233fda994150df1da3d0b573402f20842/frontend/apps/filemanager/filemanager.lua#L1546-L1552
